skopt/parameter.py

```diff
@@ -50,59 +51,77 @@ def inverse_transform(self, values):
         return np.exp(values)
 
 
-class CategoryTransform(TransformerMixin):
+class Distribution:
+    def transform(self, values):
+        """Transform `values` from original into warped space."""
+        return self.transformer.transform(values)
+
+    def inverse_transform(self, values):
+        """Transform `values` from warped into original space."""
+        return self.transformer.inverse_transform(values)
+
+    @abc.abstractmethod
+    def rvs(self, n_samples=None, random_state=None):
+        """
+        Sample points randomly.
```

Comment:
Change to "Randomly sample points from the original space" to explain that the samples are not in the warped space.